queries.yaml

```diff
@@ -1078,7 +1078,7 @@ post_process:
       end_zoom: 13
       items_matching: { kind: locality }
       max_items: 1
-      grid_width: 12
+      grid_width: 96
```

Comment:
Try `48` (which is 4 times original value of 12)

Reply:
Resolved in 50bb8982.